recipes/vvenc/all/test_v1_package

```diff
@@ -0,0 +1 @@
+test_package
```

Comment:
Please, remove it. It's a symbolic link to test_package. Plus, test_v1_package is no longer needed: https://github.com/conan-io/conan-center-index/discussions/17406

Reply:
https://github.com/conan-io/hooks/blob/master/hooks/conan-center.py current master branch has no KB-H073

Reply:
okay, I removed it, let's see if it passes CI.